homeassistant/components/climate/ecobee.py

```diff
@@ -259,7 +259,7 @@ def device_state_attributes(self):
         elif 'Cool' in status:
             operation = STATE_COOL
         elif 'auxHeat' in status:
-            operation = STATE_HEAT
+            operation = 'auxheat'
```

Comment:
The operations that have to be returned are standardized for the climate component. This will break it.